app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -4639,8 +4656,11 @@ describe('NetworkController', () => {
     for (const { networkType, chainId, ticker } of INFURA_NETWORKS) {
       describe(`given a type of "${networkType}"`, () => {
         it('stores the current provider configuration before overwriting it', async () => {
+          const { unrestrictedMessenger, restrictedMessenger } =
```

Comment:
Similar as above — this one is now similar to the test on line 6508.

Reply:
Removed in 2ea6a0476a7634ce97d4bef938ed7c1fc5198a67